templates/components/templates/page-layout--left-sidebar/page-layout--left-sidebar.twig

```diff
@@ -0,0 +1,11 @@
+<div class="page-layout page-layout--left-sidebar">
```

Comment:
Probably should add a variable to each of the div containers so that other's may add properties to the divs. 
Use case: As another developer I want to add the purple class to the outer wrapper and the wide class to the content div.

Reply:
WILL WORK ON THIS.